package.json

```diff
@@ -36,7 +36,7 @@
 		"test": "vitest",
 		"test:create": "node script/create-test-e2e.js",
 		"test:initialize": "node script/initialize-test-e2e.js",
-		"test:migrate": "node script/migrate-test-e2e.js",
+		"test:migrate": "vitest -r script/",
```

Comment:
[Bug] Looks like this is triggering Knip?

https://github.com/JoshuaKGoldberg/create-typescript-app/actions/runs/7313800050/job/19929116425?pr=1048

```shell
$ pnpm lint:knip

> create-typescript-app@1.50.0 lint:knip /home/runner/work/create-typescript-app/create-typescript-app
> knip

The CJS build of Vite's Node API is deprecated. See https://vitejs.dev/guide/troubleshooting.html#vite-cjs-node-api-deprecated for more details.
Unlisted dependencies (1)
script  package.json
 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

Might be a bug in Knip. You should be able to ignore it by adding `"ignoreDependencies": ["script"]` in `knip.json`. 

These little things keep popping up haha.